packages/menu/src/MenuContext.tsx

```diff
@@ -0,0 +1,12 @@
+import { createContext } from 'react';
+import { Theme } from '@leafygreen-ui/lib';
+
+interface MenuData {
+  theme: Theme;
+}
+
+export const MenuContext = createContext<MenuData>({
+  theme: Theme.Light,
+});
```

Comment:
There are probably other props we're passing in that could just be in Context, but that doesn't need to be done right now

Reply:
Yeah i was thinking about the `size` prop while I was adding this.